ocs_ci/deployment/image_registry.py

```
@@ -0,0 +1,405 @@
+        logger.info("Waiting for image registry pods to be ready")
+        ocp = OCP(kind="pod", namespace=self.namespace)
+
+        try:
+            ocp.wait_for_resource(
+                condition="Ready",
+                selector=constants.OPENSHIFT_IMAGE_SELECTOR,
+                timeout=timeout,
+                sleep=15,
+            )
+            logger.info("Image registry pods are ready")
+        except Exception as e:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)